crates/kitsune_p2p/kitsune_p2p/tests/common/test_legacy_host.rs

```diff
@@ -268,8 +268,15 @@ impl TestLegacyHost {
                                         for interval in intervals {
                                             match interval {
                                                 DhtArcRange::Bounded(lower, upper) => {
-                                                    if lower < op.location()
-                                                        && op.location() < upper
+                                                    if lower <= upper {
+                                                        if lower < op.location()
+                                                            && op.location() < upper
+                                                        {
+                                                            in_any = true;
+                                                            break;
+                                                        }
+                                                    } else if lower < op.location()
+                                                        || op.location() < upper
```

Comment:
should the bounds be inclusive?

Reply:
Yes in the first case and no in the else-if https://github.com/holochain/holochain/blob/develop/crates/holochain/src/conductor/space.rs#L431

Or to put it another way, yes it's inclusive and when describing a 'wrapping' inclusive the correct way to check that is with non-inclusive checks.